test/presubmit-tests.sh

```diff
@@ -54,10 +54,9 @@ install-node() {
 }
 
 ui-unittest(){
-    install-node
-    npm clean-install
+  install-node
 
-  CI=true npm test || {
+  make ui-test || {
```

Comment:
why are we removing `CI=true` ?

Reply:
I have added that in the specified make target
For e.g.
```
# Make target for UI Unit Test
.PHONY: ui-test
ui-test:
        @echo "----------------------------"
        @echo "-- Running UI Unit Tests --"
        @echo "----------------------------"
        cd ui && npm clean-install && CI=true npm test
```